opencti-platform/opencti-front/src/private/components/import/ImportContent.jsx

```diff
@@ -652,70 +598,11 @@ class ImportContentComponent extends Component {
               </Form>
             )}
           </Formik>
-          <Formik
-            enableReinitialize={true}
-            initialValues={{ name: '', labels: [] }}
-            validationSchema={fileValidation(t)}
-            onSubmit={this.onSubmitCreate.bind(this)}
-            onReset={this.onResetCreate.bind(this)}
-          >
-            {({ submitForm, handleReset, isSubmitting }) => (
-              <Form>
-                <Dialog
-                  PaperProps={{ elevation: 1 }}
-                  open={displayCreate}
-                  onClose={this.handleCloseCreate.bind(this)}
-                  fullWidth={true}
-                >
-                  <DialogTitle>{t('Create a workbench')}</DialogTitle>
-                  <DialogContent>
-                    <Field
-                      component={TextField}
-                      variant="standard"
-                      name="name"
-                      label={t('Name')}
-                      fullWidth={true}
-                    />
-                    <Field
-                      component={AutocompleteFreeSoloField}
-                      style={{ marginTop: 20 }}
-                      name="labels"
-                      multiple={true}
-                      textfieldprops={{
-                        variant: 'standard',
-                        label: t('Labels'),
-                      }}
-                      options={[]}
-                      renderOption={(optionProps, option) => (
-                        <li {...optionProps}>
-                          <div className={classes.icon}>
-                            <ItemIcon type="Label" />
-                          </div>
-                          <div className={classes.text}>{option.label}</div>
-                        </li>
-                      )}
-                      classes={{
-                        clearIndicator: classes.autoCompleteIndicator,
-                      }}
-                    />
-                  </DialogContent>
-                  <DialogActions>
-                    <Button onClick={handleReset} disabled={isSubmitting}>
-                      {t('Cancel')}
-                    </Button>
-                    <Button
-                      type="submit"
-                      color="secondary"
-                      onClick={submitForm}
-                      disabled={isSubmitting}
-                    >
-                      {t('Create')}
-                    </Button>
-                  </DialogActions>
-                </Dialog>
-              </Form>
-            )}
-          </Formik>
+          <WorkbenchFileCreator
+            handleCloseCreate={this.handleCloseCreate.bind(this)}
+            openCreate={!!displayCreate}
```

Comment:
Why is !! needed ? Isn't that a boolean ?

Reply:
That must be simply IntelliJ yelling because you are in a javascript and not typescript file.